data/json/items/containers.json

```diff
@@ -2718,6 +2719,172 @@
     ],
     "qualities": [ [ "CONTAIN", 1 ] ]
   },
+  {
+    "id": "aquarium_medium_drying",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "50 gallon aquarium (drying)", "str_pl": "50 gallon aquariums (drying)" },
+    "description": "A 50 gallon glass aquarium, used for safely storing small animals for use as pets or food.  This one has been freshly made and requires time for the silicone to dry.",
+    "weight": "45 kg",
+    "volume": "222 L",
+    "longest_side": "91 cm",
+    "price": 5000,
+    "price_postapoc": 750,
+    "bashing": 1,
+    "material": [ "glass" ],
+    "symbol": ")",
+    "color": "light_cyan",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": false,
+        "open_container": true,
+        "max_contains_volume": "189 L",
+        "max_contains_weight": "300 kg"
+      }
+    ],
+    "qualities": [ [ "CONTAIN", 1 ] ],
+    "use_action": {
+      "target": "aquarium_medium",
+      "msg": "You tear off the tape from the silicone, looks like the aquarium is ready.",
```

Comment:
Aquariums are checked by filling them with water, not just by examining the seams. Also, what tape?

Reply:
You are right, that is a way that they should be checked. However, I am not able to code it in such a way that the player has to fill the aquarium with water every time they check whether it is ready or not. The tape is duct tape used in the crafting recipes, I was not going to add a new type of tape solely for aquarium crafting.